pkgs/by-name/sa/saber/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter,
+  gst_all_1,
+  libunwind,
+  orc,
+  webkitgtk_4_1,
+}:
+flutter.buildFlutterApplication rec {
+  pname = "saber";
+  version = "0.25.1";
+
+  src = fetchFromGitHub {
+    owner = "saber-notes";
+    repo = "saber";
+    rev = "v${version}";
```

Comment:
Prefer prefixing tags with `refs/tags/` in order to avoid ambiguities with branches (see #162454)